packaging/linux/amazon-cloudwatch-agent.spec

```diff
@@ -10,6 +10,8 @@ Source:     amazon-cloudwatch-agent.tar.gz
 
 %define _enable_debug_packages 0
 %define debug_package %{nil}
+%define _source_payload w9.xzdio
```

Comment:
For testing, set this to 0, so build is faster bc compression takes a long time when set to 9